TUTORIAL.md

```diff
@@ -0,0 +1,100 @@
+# Getting started
+Before you start, make sure you understand the concept or database migration and rollback.
+
+Create new Go project and configure a database for it. Make sure that your database is supported [here](README.md#databases)
+For the purpose of this tutorial let's create PostgreSQL database called `example`.
+Our user here is `postgres`, and host is `localhost`.
```

Comment:
Also mention what the password is. I'd use a password that's distinct from the username for clarity

Reply:
Ok